### PR TITLE
core: models: add metrics

### PR DESCRIPTION
**Pull request by amrohassaan**
add metrics attrs and use it in report to call the new metric ep in
Squad.

### CODE REVIEW

**Comment by amrohassaan** [comment]
Add summary per env as well after the metadata.
summary per env

**Comment by chaws** [comment]
Hey @amrohassaan tests seem to be failing, could you review that?

**Comment by amrohassaan** [comment]
@chaws  yes I know, fixing them today.

File: examples/build_report.py [hunk 1]
```diff
@@ -57,19 +57,45 @@
...
+envs_summaries = {}
+testruns = {}
 for env in squad_envs.values():
     suites = {}
+    test_runs = set()
     envs.update({env.slug: suites})
+    envs_summaries.update({env.slug: {'pass': 0, 'fail': 0, 'xfail': 0, 'skip': 0}})
     tests = Squad().tests(count=ALL, test_run__environment=env.id, test_run__build=build.id)
+    metrics = Squad().metrics(count=ALL, test_run__environment=env.id, test_run__build=build.id)
     # group tests by suite
```
Comment (chaws) [inline examples/build_report.py:69]: Do you mind removing this comment?
```python
# group tests by suite
```

**Comment by amrohassaan** [comment]
@chaws I've removed the comment you asked for.

**Comment by amrohassaan** [comment]
@chaws fixed indentation and pushed.

**Comment by chaws** [comment]
It's failing a test, but I'll merge this now and fix it in a newer commit. Also, table strings look weird in chrome but fine in Firefox, we can fix this this week.

**Pull request merged**